components/activecampaign/activecampaign.app.mjs

```diff
@@ -0,0 +1,450 @@
+import { axios } from "@pipedream/platform";
+import inflection from "inflection";
+import constants from "./common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "activecampaign",
+  propDefinitions: {
+    eventType: {
+      type: "string",
+      label: "Event Type",
+      description:
+        "Emit events for the selected event type. See the official docs for more information on event types. https://developers.activecampaign.com/page/webhooks",
+      async options({ page }) {
+        if (page) {
+          return [];
+        }
+        const { webhookEvents } = await this.listWebhookEvents();
+        return webhookEvents.map((e) => ({
+          label: inflection.humanize(e),
+          value: e,
+        }));
+      },
+    },
+    sources: {
+      type: "string[]",
+      label: "Sources",
+      description:
+        "The sources causing an event to occur. Leave blank to include all sources.",
+      optional: true,
+      default: [],
+      options: constants.ALL_SOURCES,
```

Comment:
This default value isn't needed. If nothing is selected, the value will already be `[]`.
```suggestion
      optional: true,
      options: constants.ALL_SOURCES,
```